src/github_runner_image_builder/templates/cloud-init.sh.j2

```diff
@@ -67,9 +67,8 @@ function configure_system_users() {
     /usr/bin/id -u ubuntu &>/dev/null || useradd --create-home ubuntu
     echo "PATH=\$PATH:/home/ubuntu/.local/bin" >> /home/ubuntu/.profile
     echo "PATH=\$PATH:/home/ubuntu/.local/bin" >> /home/ubuntu/.bashrc
-    /usr/sbin/groupadd -f microk8s
     /usr/sbin/groupadd -f docker
-    /usr/sbin/usermod --append --groups docker,microk8s,lxd,sudo ubuntu
```

Comment:
Should these be here even if microk8s is not in the image?

Reply:
You're right, nice catch :)